solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -9,11 +9,24 @@ import "../../../token/interfaces/ISmartToken.sol";
   * The liquidity pool v1 converter is a specialized version of a converter that manages
   * a classic bancor liquidity pool.
   *
-  * Even though classic pools can have many reserves, the most common configuration of
-  * the pool has 2 reserves with 50%/50% weights.
+  * Even though pools can have many reserves, the standard pool configuration
+  * is 2 reserves with 50%/50% weights.
 */
 contract LiquidityPoolV1Converter is LiquidityPoolConverter {
+    struct Fraction {

```

Comment:
We probably wanna move this `struct Fraction` to contract `BaseConverter` (or even to a library which contract `BaseConverter` will be using).

Reply:
moved into a dedicated `Types` utility contract